utils/language.ts

```diff
@@ -31,12 +31,16 @@ export function matchLanguages(languages: string[], acceptLanguages: readonly st
 
 export function getLanguageForRecs(lang: string): string {
   const availableLocales = [
-    'fr', 'fr-FR',
-    'es', 'es-ES',
-    'it', 'it-IT',
-    'en', 'en-CA', 'en-GB', 'en-US',
-    'de', 'de-DE', 'de-AT', 'de-CH',
+    'de-DE',
+    'es-ES',
+    'it-IT',
+    'fr-FR',
+    'en-US',
+    'en-CA',
+    'en-IN',
+    'en-GB',
+    'en-IE',
   ]
 
-  return availableLocales.includes(lang) ? lang : 'en'
+  return availableLocales.includes(lang) ? lang : 'en-US'
```

Comment:
Updated this file to match the supported languages provided by the platform team